### PR TITLE
zabbix_hostmacro module updated: add description property for Host macro

### PR DESCRIPTION
**Pull request by ndf-dw**
##### SUMMARY

Add description property for Host macro creation/update: allow to set/update description of Zabbix host macros.

##### ISSUE TYPE

- Feature Pull Request

##### COMPONENT NAME

zabbix_hostmacro module

##### ADDITIONAL INFORMATION

This Add 'description' property for Host Macro module (https://docs.ansible.com/ansible/latest/collections/community/zabbix/zabbix_hostmacro_module.html)

It uses 'description' property described in: https://www.zabbix.com/documentation/current/en/manual/api/reference/usermacro/object#host-macro

### CODE REVIEW

**Comment by BGmot** [comment]
Please add changelog fragment.

**Review by pyrodie18 — CHANGES_REQUESTED** [review]
Change Fragment

File: changelogs/fragments/hostmacro_modules.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+minor_change:
+  - zabbix_hostmacro module - Add description property for Host macro creation/update: allow to set/update description of Zabbix host macros.
```
Comment (pyrodie18) [inline changelogs/fragments/hostmacro_modules.yml:2]: This file failed during sanity checks, I think because of the colon you have in here.

**Review by pyrodie18 — CHANGES_REQUESTED** [review]
See comments

File: changelogs/fragments/hostmacro_modules.yml (hunk 1 above)
Comment (BGmot) [reply changelogs/fragments/hostmacro_modules.yml:2]: yes you can't use colons here.

**Review by pyrodie18 — APPROVED** [review]

**Comment by pyrodie18** [comment]
Great work

**Pull request merged**